cli/test/smokehouse/test-definitions/perf-trace-elements.js

```diff
@@ -13,6 +13,10 @@ const config = {
     // webpages present here, hence the inclusion of 'best-practices'.
     onlyCategories: ['performance', 'best-practices'],
 
+    // BF cache will request the page again, initiating additional network requests.
+    // Disable the audit so we only detect requests from the normal page load.
+    skipAudits: ['bf-cache'],
```

Comment:
Maybe we should have a test-definition base config we extend from? we could also specify `only-categories: []` in it, forcing smoke configs that extend it to be more specific (I assume that works...)

Reply:
Seems like overkill plus it would be confusing when writing a test to bounce between the smokehouse base config and the test specific config.

This issue is only present on a handful of audits.